src/commands/synthetics/__tests__/reporters/default.test.ts

```diff
@@ -35,22 +36,23 @@ describe('Default reporter', () => {
   const reporter = new DefaultReporter(mockContext as {context: BaseContext})
 
   it('should log for each hook', () => {
-    type ReporterCall = {[Fn in keyof MainReporter]: [Fn, Parameters<MainReporter[Fn]>]}[keyof MainReporter]
+    type ReporterCall = {[Fn in keyof MainReporter]: [Fn, Parameters<MainReporter[Fn]>, number]}[keyof MainReporter]
 
     // `testWait`/`resultReceived` is skipped as nothing is logged for the default reporter.
     const calls: ReporterCall[] = [
-      ['error', ['error']],
-      ['initErrors', [['error']]],
-      ['log', ['log']],
-      ['reportStart', [{startTime: 0}]],
-      ['resultEnd', [getApiResult('1', getApiTest()), '']],
-      ['runEnd', [getSummary(), '']],
-      ['testTrigger', [getApiTest(), '', ExecutionRule.BLOCKING, {}]],
-      ['testsWait', [[getApiTest()]]],
+      ['error', ['error'], 1],
+      ['initErrors', [['error']], 1],
+      ['log', ['log'], 1],
+      ['reportStart', [{startTime: 0}], 1],
+      ['resultEnd', [getApiResult('1', getApiTest()), ''], 1],
+      ['runEnd', [getSummary(), ''], 1],
+      ['testTrigger', [getApiTest(), '', ExecutionRule.BLOCKING, {}], 1],
+      ['testsWait', [[getApiTest()], '', ''], 2],
```

Comment:
`testsWait` now has 2 calls of `context.stdout.write()`:
- One to print the summary link.
- One to show the `ora` spinner.

We can't merge this text into a multiline text (like `'View summary...\n\nWaiting for 1 test result...'`) because of how the spinner works.

While the spinner spins, **only the last line** is shown and frequently rewritten. Only when the spinner stops, the full multiline text reappears.

I don't think it was a hard requirement to have a _**single**_ `context.stdout.write()` call per reporter method.